routes/api/v1/meals.js

```diff
@@ -0,0 +1,35 @@
+var express = require('express');
+var router = express.Router();
+var pry = require("pryjs")
+var Meal = require("../../../models").Meal;
+var Food = require("../../../models").Food;
+var MealFood = require("../../../models").MealFood;
+
+
+router.get("/", function(req, res) {
+  Meal.findAll({
+
+  include: [
+    {
+    model: Food,
+    as: 'foods',
+    attributes: ["id", "name", "calories"],
+    through: {
+      attributes: []
+    }
+    }]
```

Comment:
So, adding the through and setting the attributes as empty removes the object. That's a cool workaround.

Reply:
Yes! Saw it in the comments sections here: https://gist.github.com/zcaceres/83b554ee08726a734088d90d455bc566